typescript/sdk/src/consts/environments/index.ts

```diff
@@ -3,32 +3,30 @@ import { types } from '@hyperlane-xyz/utils';
 import { LooseChainMap } from '../../types';
 import { objMap } from '../../utils/objects';
 
-import mainnet2 from './mainnet2.json';
 import mainnet from './mainnet.json';
 import test from './test.json';
-import testnet2 from './testnet2.json';
-import testnet3 from './testnet3.json';
+import testnet from './testnet.json';
 
-export const environments = { test, testnet2, mainnet, testnet3, mainnet2 };
+export const environments = { test, testnet, mainnet };
 
 type HyperlaneCoreAddressMap = LooseChainMap<{
   mailbox: types.Address;
   multisigIsm: types.Address;
   interchainGasPaymaster: types.Address;
-  interchainAccountRouter: types.Address;
-  interchainQueryRouter: types.Address;
+  //interchainAccountRouter: types.Address;
+  //interchainQueryRouter: types.Address;
```

Comment:
Should we leave them in but set them to null in order to avoid changes to the type?
Also should there be a TODO or something to renable these soon?

Reply:
Yeah I'm fine with making them optional for now